webapp/src/components/modals/channel_settings/edit_channel_settings.tsx

```diff
@@ -277,6 +293,21 @@ export default class EditChannelSettings extends PureComponent<Props, State> {
             component = <Loading/>;
         }
 
+        const {showConfirmModal} = this.state;
+        const confirmComponent = (
+            <ConfirmModal
+                cancelButtonText={'Cancel'}
+                confirmButtonText={'Delete'}
+                confirmButtonClass={'btn btn-danger'}
+                hideCancel={false}
+                message={'Delete Subscription ?'}
+                onCancel={this.handleDeactivateCancel}
+                onConfirm={this.handleConfirmDelete}
+                show={showConfirmModal}
+                title={'Subscription'}
```

Comment:
I'm curious if "Subscription" is the best UX here. I'm 0/5 on changing it to "Delete Subscription". @asaadmahmood Is there an established convention for this?